mkosi.images/system/mkosi.conf.d/10-debian-ubuntu/mkosi.postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+# By default Suggests are not installed (and often Recommends are disabled too), which means we will miss
+# the dlopen optional dependencies, but the tests need them, so parse them from the package metadata and
+# install them.
+
+# apt needs network access to install packages
+if [ "$1" = "build" ] || [ "$WITH_NETWORK" -eq 0 ]; then
```

Comment:
I would drop the WITH_NETWORK check it shouldn't hurt to always do this.

Maybe gate it on NO_BUILD instead.

Reply:
but apt fails if WITH_NETWORK=0, and the script fails then? Also this is needed for both local builds and package installs, so it needs to run regardless of NO_BUILD

Reply:
Ah I forgot that now the build image is the same as the runtime one, ok then it can be done